manifests/11_service_monitor.yaml

```diff
@@ -7,7 +7,18 @@ metadata:
     name: marketplace-operator
 spec:
   endpoints:
-  - port: metrics
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      metricRelabelings:
+        - action: drop
```

Comment:
Hey, landed here only because I need to do the same thing to a metric endpoint of mine. I suspect this relabeling section is not necessary, only the scheme part.